Source/carthage/Build.swift

```diff
@@ -31,8 +31,16 @@ public struct BuildCommand: CommandType {
 
 				let (stdoutSignal, schemeSignals) = self.buildProjectInDirectoryURL(directoryURL, options: options)
 				stdoutSignal.observe { data in
+
 					stdoutHandle.writeData(data)
-					stdoutHandle.synchronizeFile()
+
+					switch stdoutHandle {
```

Comment:
Hm, I don't like having to introspect this value.

Could `openTemporaryLogFile` give us a `FILE *` instead? Then we can treat them all the same.